### DISCUSSION TITLE
DAI 2024-2025 - My Docker Compose application - Léon Surbeck

### DISCUSSION THREAD

**Discussion by Bananonymous** [heig-vd-dai-course/heig-vd-dai-course#408] [Show and tell]
![image](https://github.com/user-attachments/assets/e6d1bfff-6192-41bc-9ccb-54591038aac4)
[Here !](https://github.com/Bananonymous/DAI-05-DockerJavaIOs)

**Comment by ludelafo** [comment]
Thank you!

This only thing I see that could be improved is the entrypoint/command of your Dockerfile to make it more versatile.

You can always find the solution in the solution repository if you ever need to compare your work with the official one here: https://github.com/heig-vd-dai-course/heig-vd-dai-course-solutions.

Do not hesitate to ask questions if something is not clear, either in this discussion or in the Q&A category of the GitHub Discussions here: https://github.com/orgs/heig-vd-dai-course/discussions/categories/q-a.
